### PR TITLE
Feature/mobile guest page

### PR DESCRIPTION
**Pull request by Charanrajcdw**

### CODE REVIEW

File: src/containers/GuestsContainer/GuestsContainer.tsx [hunk 1]
```diff
@@ -1,25 +1,95 @@
...
+  const prevSlide = () => {
+    splideRef.current?.go('<');
+  };
+
+  const nextSlide = () => {
+    splideRef.current?.go('>');
+  };
+
+  const getSlides = () => {
+    return guests.map((guest, index) => {
```
Comment (laxman1522) [inline src/containers/GuestsContainer/GuestsContainer.tsx:30]: guests?.map

File: src/containers/GuestsContainer/GuestsContainer.tsx [hunk 2]
```diff
@@ -1,25 +1,95 @@
...
+      return (
+        <SplideSlide key={index}>
+          <div className="h-full rounded-[40px] overflow-hidden">
+            <Image
+              src={guest.imageUrl}
```
Comment (laxman1522) [inline src/containers/GuestsContainer/GuestsContainer.tsx:35]: use null check ? here

File: src/containers/GuestsContainer/GuestsContainer.tsx [hunk 3]
```diff
@@ -1,25 +1,95 @@
...
+          <div className="mt-16 mb-12">
+            <Splide
+              onMove={(splideInstance: any, newIndex: number) => {
+                setActiveSlide(newIndex);
+              }}
+              onMounted={(splide: any) => {
+                splideRef.current = splide;
+              }}
+              aria-label={GUESTS.ourGuests}
+              options={{
```
Comment (laxman1522) [inline src/containers/GuestsContainer/GuestsContainer.tsx:69]: move options to constants or configuration file

File: src/utils/data-utils.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import guestsData from '../../data/guest_details.json';
+
+export const getAllGuests = () => {
+  const allGuests = guestsData.guests;
+  allGuests.sort((a, b) => {
```
Comment (laxman1522) [inline src/utils/data-utils.tsx:5]: use null check ?

Comment (Charanrajcdw) [reply src/utils/data-utils.tsx:5]: added empty array guestsData.guests || []

File: src/containers/GuestsContainer/GuestsContainer.tsx (hunk 3 above)
Comment (Charanrajcdw) [reply src/containers/GuestsContainer/GuestsContainer.tsx:69]: moved the options to constants

File: src/containers/GuestsContainer/GuestsContainer.tsx (hunk 1 above)
Comment (Charanrajcdw) [reply src/containers/GuestsContainer/GuestsContainer.tsx]: added them

**Pull request merged**